packages/cdk8s-plus-17/test/deployment.test.ts

```diff
@@ -1,6 +1,15 @@
 import { Testing } from 'cdk8s';
+import { Node } from 'constructs';
 import * as kplus from '../src';
 
+test('defaultChild', () => {
```

Comment:
how do we enforce that such a test exists for all L2s? (for later)

Reply:
Marked for later